src/slices/AccountSlice.ts

```diff
@@ -156,16 +159,28 @@ export const getBalances = createAsyncThunk(
 export const getDonationBalances = createAsyncThunk(
   "account/getDonationBalances",
   async ({ address, networkID, provider }: IBaseAddressAsyncThunk) => {
-    const mockSohmContract = new ethers.Contract(addresses[networkID].MOCK_SOHM as string, MockSohm, provider);
-    // Change to .allowance before mainnet
-    const giveAllowance = await mockSohmContract._allowedValue(address, addresses[networkID].GIVING_ADDRESS);
+    /*
+      On testnet it's been best for testing Give to use a pseudo-sOHM contract
+      that gives us more control to rebase manually when needed. However, this 
+      makes it not as perfectly translatable to mainnet without changing any parameters
+      this is the best way to avoid manually switching out code every deployment
+    */
+    let giveAllowance;
+    if (networkID === 1) {
```

Comment:
Do we need to support chain IDs other than 1 and 4?

Reply:
might need an else statement to handle arbitrum and avalanche, yes, not sure what we should output there. Just 0's?

Reply:
think I'll just change it to 
`
let giveAllowance = 0;
if (networkID ===......
`

and leave off an else statement until we take Give cross-chain. Thoughts?